include_physics/ideal_MHD.h

```diff
@@ -663,10 +663,59 @@ inline void add_surface_terms(REAL time, uint ix, uint iy, uint iz, const global
   }
 #endif
 
+#elif defined USE_BOUNDARY_FLUX_LFF // USE_BOUNDARY_FLUX_HLL -> USE_BOUNDARY_FLUX_HLL
+  REAL fluxm[NUM_CONSERVED_VARS] = {0.0}; // The Lax-Friedrichs Flux, named after Kurt Friedrichs and Peter Lax
```

Comment:
Many people tend to write the full name "Kurt Otto Friedrichs" because they like the abbreviation "KO Friedrichs" :wink: